Vagrantfile

```diff
@@ -1,18 +1,30 @@
 Vagrant.require_version ">= 1.8.0"
 
 Vagrant.configure(2) do |config|
+  config.vm.box = "opensuse/Leap-15.4.x86_64"
+  config.vm.define "machine1"
 
-  config.vm.box = "opensuse/Leap-15.3.x86_64"
   config.vm.provision "ansible" do |ansible|
     # ansible.verbose = "v"
     ansible.playbook = "playbook.yml"
+    ansible.groups = {
+      "trento-server" => ["machine1"],
+      "postgres-hosts" => ["machine1"],
+      "prometheus-hosts" => ["machine1"],
+      "rabbitmq-hosts" => ["machine1"]
+    }
     ansible.extra_vars = {
       web_postgres_password: "pass",
       wanda_postgres_password: "wanda",
       rabbitmq_password: "trento",
+      nginx_vhost_filename: "trento.conf",
       prometheus_url: "http://localhost",
       web_admin_password: "adminpassword",
-      trento_server_name: "trento.local trento.local:8080"
+      trento_server_name: "trento.local trento.local:8080",
+      nginx_ssl_cert_as_base64: "false",
```

Comment:
Do you need to put them with the default value?

Reply:
No, it was a reminder for me, but we can remove them